doc/classes/TileData.xml

```diff
@@ -37,6 +37,20 @@
 				Returns how many polygons the tile has for TileSet physics layer with index [code]layer_id[/code].
 			</description>
 		</method>
+		<method name="get_constant_angular_velocity" qualifiers="const">
+			<return type="float" />
+			<argument index="0" name="layer_id" type="int" />
+			<description>
+				Returns the constant angular velocity.
```

Comment:
And this could be more explicit too in order to avoid any confusion:
```suggestion
				Returns the constant angular velocity applied to objects colliding with this tile.
```